tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -259,7 +268,7 @@ install_cc_image() {
 	install_cached_component \
 		"${component}" \
 		"${jenkins}" \
-		"${osbuilder_last_commit}-${guest_image_last_commit}-${agent_last_commit}-${libs_last_commit}-${attestation_agent_version}-${gperf_version}-${libseccomp_version}-${pause_version}-${rust_version}-${image_type}-${AA_KBC}" \
+		"${osbuilder_last_commit}-${guest_image_last_commit}-${agent_last_commit}-${libs_last_commit}-${attestation_agent_version}-${gperf_version}-${libseccomp_version}-${pause_version}-${rust_version}-${image_type}-${AA_KBC}${initramfs_version}" \
```

Comment:
We want `initramfs_version`, which is a misleading name, to be `initramfs_image_last_commit`, and I'd add it just after the `guest_image_last_commit`.

Reply:
Changed the order.